git-merge-conflict.md

```diff
@@ -0,0 +1,52 @@
+1. **Pull the recipe file**: Once the recipe file has been created and committed, everyone else in your group should `git pull` that commit to their local copy of the repository.
+1. **Find your recipe scrap**: Each group is assigned six recipe scraps that should all be part of the same recipe. You can split these scraps up amongst your group however you wish. The scraps assigned to each group are [listed in the carnival-recipes repo](https://github.com/Ada-C7/carnival-recipes/#recipe-scraps-for-each-group).
+1. **Add your recipe scrap**: Copy the contents of your recipe scrap into the recipe file and delete the recipe scrap then commit both of these changes.
+
+**Do not** push your changes to GitHub yet! If you're finished early with this step help your groupmates.
+
+### Building the recipe
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together. There are a number of ways that your group can accomplish that, and you should discuss them to figure out which strategy to employ:
+
+- **Round robin**: This strategy involves direct coordination between everyone in the group, resulting in a strictly ordered merge sequence.
+  - One at a time each person pushes their changes and resolves any merge conflicts.
+  - The next person in turn pulls those changes and pushes their own until everyone's changes have been pushed.
+- **Ad-hoc**: This approach relies the most upon Git's merge coordination capabilities to ensure that everyone's changes get merged in properly.
+  - Essentially, everyone attempts to `git push` at once.
+  - The first person to get there doesn't have to do a merge, but everyone after that probably will.
+- **Branched**: This approach is kind of a mix of the other two.
+  - Within your group split off into pairs.
+  - Both people in the pair create a branch _with the same name_. You can create a branch with `git checkout -b <branch name>`.
+  - Both people in the pair push their changes to the branch (`git push origin <branch name>`).
+  - Whoever ends up pushing second will have to merge in the other person's changes.
+  - After merging the pair's changes together, merge the result into the `master` branch:
+    - `git checkout master`
+    - `git merge <branch name>`
+  - Push the `master` branch up to GitHub.
+  - The first pair to complete this won't have to do any more merging, but the other two pairs will.
+
+Once your recipe file has been completed everyone should review it for completeness. Make sure that none of the lines from your scrap are missing from the final result.
+
+If everyone agrees that the recipe looks correct one person should submit a PR to the original [carnival-recipes](https://github.com/Ada-C7/carnival-recipes) repo.
```

Comment:
It feels like when we have to update this for the next cohort, we should have the repo linked in one place rather than many so it is easier to update when that needs to happen. We can also remove the links completely and associate the repo in the calendar per cohort rather than in this document so it's always applicable for each cohort.